ts-microservices-lambda/test/pulumi/index.ts

```diff
@@ -0,0 +1,45 @@
+import * as aws from "@pulumi/aws";
+import * as awsx from "@pulumi/awsx";
+import { PolicyPack, validateResourceOfType, StackValidationPolicy } from "@pulumi/policy";
+
+const lambdaPolicy: StackValidationPolicy = {
+    name: "lambda-test",
+    description: "Lambda integration tests.",
+    enforcementLevel: "mandatory",
+    validateStack: async (args, reportViolation) => {
+        const lambdas = args.resources.filter(r => r.isType(aws.lambda.Function));
+        if (lambdas.length !== 2) {
+            reportViolation(`Expected one lambda function but found ${lambdas.length}`);
+            return;
+        }
+
+        const lambda = lambdas[0].asType(aws.lambda.Function)!;
```

Comment:
minor: of-each to check both?

Reply:
no since were just doing defaults i think leaving it and checking one should be ok for now